server/src/core/server/services/users/delete.ts

```diff
@@ -309,7 +313,11 @@ async function deleteUserComments(
     },
     GQLCOMMENT_STATUS.REJECTED,
     now,
-    isArchived
+    isArchived,
+    {
+      code: GQLREJECTION_REASON_CODE.OTHER,
+      detailedExplanation: "User account deleted",
```

Comment:
We can translate this using the following:

```
import { I18n, translate } from "coral-server/services/i18n";
...

const text = "User account deleted";
const key = "common-userDeletion-userAccountDeleted";
const args = {};

const bundle = this.i18n.getBundle(tenant.locale);
const result = translate(bundle, text, key, args);
```

Then the translations for this can be specified in `common.ftl` under `server/src/core/server/lcoales/enu-US`.